### PR TITLE
Add basic React components to 3ID-UI

### PR DESCRIPTION
**Pull request by Sterahi**
Screen Shot 2021-09-22 at 1 11 58 PM
Above is the basic display of the Modal. Still need logo from Sena or we can remove & rebrand slightly. 

Please ignore any files outside of `apps/3id-ui`. There were issues with packages and I've merged develop to try and sort it out.

### CODE REVIEW

File: apps/3id-ui/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        }}>
+        Accept
+      </div>
+    )
+    if (declineNode) {
+      declineNode = (
+        <div
+          className="btn"
+          onClick={() => {
+            resolve(true)
```
Comment (PaulLeCam) [inline apps/3id-ui/src/index.tsx:45]: Should resolve to `false` no?

Comment (Sterahi) [reply apps/3id-ui/src/index.tsx:45]: Totally correct, thanks!

**Review by zachferland — COMMENTED** [review]
Overall looks good!

I merged some dependency changes in dev that should resolve lint issue here when rebased on develop. Also if you can remove the `apps/3id-ui/.eslintcache` here. Would be good to make sure you can run and build on node16 as well, we should probs align on versions, but CI uses that right now.

Looks like the two high level things that might be missing still are:

1) handlings errors on error notification, this is just really display errors to user 
2) closing the window/exiting, we are also adding better cancellation handling after, but should still be able to use the exiting close/cancel like the iframe does now 

Then we can merge and work more iteratively off the dev branch

**Review by PaulLeCam — CHANGES_REQUESTED** [review]
Looks like there's both a `package-lock.json` and a `yarn.lock` file, I think that only yarn should be used to be consistent with the rest of the repo.

**Pull request merged**